### PR TITLE
rename package & funcs

### PR DESCRIPTION
**Pull request by hopkiw**
This is a test PR for a real commit.

### CODE REVIEW

**Comment by krzyzacy** [comment]
humm I don't see github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/guestpolicy under https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/cli_tools/google-osconfig-agent?

File: main.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 	"time"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/config"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/guestpolicy"
```
Comment (krzyzacy) [inline main.go]: (it should point to this repo now)

Comment (hopkiw) [reply main.go]: lol thanks sen.. testing failures is useful for us :)

Comment (krzyzacy) [reply main.go]: it's odd that prow even cannot clone this repo due to that error message :-\

**Comment by krzyzacy** [comment]
/assign iamsubratp

^^

**Comment by hopkiw** [comment]
> /assign iamsubratp
> 
> ^^

i figured it out haha

**Comment by hopkiw** [comment]
/approved

should have happened by default since i'm an owner but we hadn't created OWNERS file yet

**Comment by hopkiw** [comment]
or
/approve

**Pull request merged**